gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -285,6 +285,10 @@ private void requestObjectMedia(OptionalInt bytesToRead) throws IOException {
   }
 
   private void cancelCurrentRequest() {
+    this.bufferedContent = null;
```

Comment:
Usually we use `this.` prefix only in constructors and setters, please remove if possible.